src/System.ComponentModel.Annotations/ref/Configurations.props

```diff
@@ -1,12 +1,7 @@
-﻿<Project>
+<Project>
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard;
```

Comment:
Have you chatted with @wtgodbe?  He’s doing some work which overlaps with this and you both may benefit from being on the same page.